client/src/PagesBody/ContestsPage/ContestsPage.js

```diff
@@ -73,26 +76,39 @@ const ContestsPage = () => {
 
   return (
     <>
-      <ContestFilterToggler
-        onClick={toggleHandler}
-        toggle={toggle}
-      ></ContestFilterToggler>
+      {useMediaQuery('(max-width:799px)') && (
```

Comment:
Maybe it would be more clear if You make a const that describes what screen is it for ?